model/flow/chain.go

```diff
@@ -169,6 +169,12 @@ var testnet = &addressedChain{
 	},
 }
 
+var bftTestNet = &addressedChain{
```

Comment:
The BFT testing framework deals with the corrupted version of our images. We isolate those images to be runnable only in a specific chain id that is exclusively dedicated for BFT testing, hence making sure that no corrupted image is utilized for any other purposes than what they are meant for, i.e., the BFT testing.

Reply:
I agree with the desired outcome. I'm just wondering if checking for `chainID != Mainnet && chainID != Testnet` is a better approach. 

For one, I think it's better to avoid adding new chain IDs unless they are really necessary - since they require code changes each time. Also, I'm wondering if we would want to use these corrupted container implementations for testing on other networks (eg. Benchnet or Canary)?